student/serializers.py

```diff
@@ -68,7 +64,6 @@ class Meta:
             'userFirstName',
             'userLastName',
             'userFullName',
-            'isAdvisor',
```

Comment:
Why is this PR deleting our changes?

Reply:
uh oh

Reply:
will a git pull fix this?

Reply:
No, I tried that. Seemed like the best option was to make a new branch with my changes